src/Microsoft.DotNet.XHarness.CLI/Program.cs

```diff
@@ -28,27 +28,18 @@ public static int Main(string[] args)
                 args[0] = "apple";

```

Comment:
The line above this also needs to be a Length > 0 as `dotnet ios` will fail without a warning as to the change.